apps/backend/src/poll/poll/poll.service.ts

```diff
@@ -87,6 +89,30 @@ export class PollService implements OnModuleInit {
     pollEvents.length && this.logger.log(`Migrated ${pollEvents.length} poll events to the new date format.`);
   }
 
+  private async migrateShowResults() {
```

Comment:
Is this the only real change to this file? I cannot tell from the diff view due to the format changes.

Reply:
With be56f71 `getParticipants` was also touched.